apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -124,6 +124,11 @@
          * Keep tracing even the backend is not available.
          */
         public static boolean KEEP_TRACING = false;
+
+        /**
+         * Should TLS enabled for gRPC channels
+         */
+        public static boolean IS_GRPC_CHANNEL_TLS_FORCED = true;
```

Comment:
Change this name to `FORCE_TLS`. There is only gRPC transport.

Reply:
This can't be open in default, because in many mode, the OAP is not open TLS.

Reply:
Yes, I agree.